docs/testplan/BGP-Scale-Test.md

```diff
@@ -0,0 +1,108 @@
+# BGP Scale Test Plan
+
+- [Overview](#overview)
+  - [Scope](#scope)
+  - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test methodology](#test-methodology)
+- [Test Cases](#test-cases)
+  - [BGP Sessions Flapping Test](BGP-Sessions-Flapping-Test)
+  - [Unisolation Test](Unisolation-Test)
+  - [Nexthop Group Member Scale Test](Nexthop-Group-Member-Scale-Test)
+
+
+## Overview
+
+This test plan is to test if control/data plane can handle the initialization/flapping of numerous BGP session holding a lot routes, and estimate the impact on it.
+
+
+## Scope
+
+This test plan runs on any device running SONIC system with fully functioning configuration with numerouse BGP peers with count 256/512.
+
+This test plan is dedicated to IPv6.
+
+This test plan shows if there is any service crush, if hardware resource run out, if device has acceptable performance and data/control plane availability.
+
+
```

Comment:
Will be better to add a section to explain the scale of the tests.

Reply:
because of the scale, it will be better to add a note that explain the implications of CPU and memory usage of cEOS neighbors, and include a link to the multi-server deployment.

Reply:
Good idea! thanks for the suggestion.

added.